src/rmt.toit

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2025 Toit contributors
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import gpio
+import rmt
+import bitmap show blit OR
+import .pixel-strip
+
+class RmtEncodingPixelStrip_ extends PixelStrip:
+  // Durations in ticks.
+  // A tick is 0.05us.
+  static T0H_ ::= 7   // 0.35 us
+  static T0L_ ::= 16  // 0.8 us
+  static T1H_ ::= 14  // 0.7 us
+  static T1L_ ::= 12  // 0.6 us
+
+  signals_/rmt.Signals
+  out_/rmt.Channel? := ?
+
+  constructor pixels/int --pin/gpio.Pin --bytes-per-pixel/int=3 --memory-block-count/int=1:
+    needed-signals := pixels * (bytes-per-pixel * 8 * 2)
+    needed-memory := needed-signals * rmt.Signals.BYTES-PER-SIGNAL
+    out_ = rmt.Channel --output
+        pin
+        --memory-block-count=memory-block-count
+        --clk-div=4  // The clock is at 80MHz. so dividing by 4 gives 0.05us ticks.
+        --idle-level=0
+    signals_ = rmt.Signals.from-bytes (ByteArray.external needed-memory)
+
+    super pixels --bytes-per-pixel=bytes-per-pixel
+
+  close -> none:
```

Comment:
Do you want this to have a finalizer?

Reply:
I don't think that's necessary. The RMT peripheral should have its own finalizer.